services/has-many-getter.js

```diff
@@ -47,6 +47,25 @@ function HasManyGetter(model, association, opts, params) {
           resolve(record);
         });
       });
+    })
+    .then(function(records) {
+      if (params.sort) {
+        let fieldSort = params.sort;
+        let descending = false;
```

Comment:
I prefer to avoid ECMA6 feature here to be sure old node version project can use our package.